src/containers/Application.js

```diff
@@ -446,23 +440,13 @@ class Application extends Component {
 		}
 	}
 
-	// First argument is href to page
-	// Second (optional) is string that is icon from core-ui icons
-	// Third (optional) is target for link
-	addHelpButton(url, icon = "cil-info", target) {
+	addHelpButton(path) {
 		useEffect(() => {
-			this.Store.dispatch({
-				type: CHANGE_HELP_URL,
-				payload: { url, icon, target }
-			})
+			this.HelpService.setData(path);
 			return () => {
```

Comment:
@Aringoaway Why do you need to return a Store dispatch? I thought that `HelpService` will take care of that. From my point of view are those lines https://github.com/TeskaLabs/asab-webui/blob/3ddb565f5af9e7b5cac1f77c9366ee1485886e2e/src/containers/Application.js#L446-L451 redundant

Reply:
And should be moved to the service, if needed